server/models/meal.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const mealSchema = new Schema({
+  title: { type: String, required: true },
+  picURL: String,
+  price: { type: Number, required: true },
+  // ie: 2 people, 4 people
+  servingSize: { type: String, required: true },
+  // ie: family style, buffet
+  servingType: String,
+  cuisineType: [{ type: String, required: true }],
+  ingredients: { type: String, required: true },
+  createdAt: { type: Date, default: Date.now },
+  updatedAt: Date,
+  chefId: {type: Schema.Types.ObjectId, ref: 'Chefs'}
```

Comment:
I'm thinking that we probably only need either the meals array on the Chef model, or the chefId on the Meal model, but not both?
Is there any possibility that two chefs would have the same meal? (I'm thinking not)

Reply:
I think for now we can just do the chefid on the meal model. I would suppose they could have the same meal or even title.